internal/net/call/trace.go

```diff
@@ -20,7 +20,7 @@ import (
 	"go.opentelemetry.io/otel/trace"
 )
 
-const traceHeaderLen = 25
+const traceHeaderLen = 25 // handler_key + deadline + trace_context
```

Comment:
It would be nice if all of the header constants that depend on each other were described next to each other in the same constant block

Reply:
Removed